addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -202,7 +202,7 @@ function dragNDrop(snippet, position = "bottom") {
         trigger: `#oe_snippets .oe_snippet[name="${snippet.name}"] .oe_snippet_thumbnail:not(.o_we_already_dragging)`,
         extra_trigger: "body.editor_enable.editor_has_snippets",
         moveTrigger: '.oe_drop_zone',
-        content: _t(`Drag the <b>${snippet.name}</b> building block and drop it in your page.`),
```

Comment:
@qsm-odoo does this work now? Translating template strings?

Reply:
No, you should use `_.str.sprintf` and if you add translatable terms in standard, don't forget to update the `pot` file :wink:

Reply:
Well, the tour_utils file uses this all over so that probably needs to be fixed, as you can see this commit isn't the one that introduced this use of `_t`.

Reply:
Indeed, maybe @younn-o can fix all of that in the same commit and updating the .pot file. That way that will give a good excuse for actually changing the sentence in a stable version ;)